src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1375,46 +1217,39 @@ bool JasmineGraphInstanceService::sendFileThroughService(std::string host, int d
     bcopy((char *)server->h_addr, (char *)&serv_addr.sin_addr.s_addr, server->h_length);
     serv_addr.sin_port = htons(dataPort);
     if (Utils::connect_wrapper(sockfd, (struct sockaddr *)&serv_addr, sizeof(serv_addr)) < 0) {
-        std::cerr << "ERROR connecting to port " << dataPort << std::endl;
+        instance_logger.error("ERROR connecting to port " + std::to_string(dataPort));
         return false;
     }
 
-    int result_wr = write(sockfd, fileName.c_str(), fileName.size());
-
-    if (result_wr < 0) {
-        instance_logger.log("Error writing to socket", "error");
-    }
-
-    bzero(data, INSTANCE_DATA_LENGTH + 1);
-    read(sockfd, data, INSTANCE_DATA_LENGTH);
-    string response = (data);
-    response = Utils::trim_copy(response, " \f\n\r\t\v");
+    Utils::send_str_wrapper(sockfd, fileName);
 
+    string response = Utils::read_str_trim_wrapper(sockfd, data, INSTANCE_DATA_LENGTH);
     if (response.compare(JasmineGraphInstanceProtocol::SEND_FILE) == 0) {
-        std::cout << "Sending file " << filePath << " through port " << dataPort << std::endl;
-
         FILE *fp = fopen(filePath.c_str(), "r");
         if (fp == NULL) {
             instance_logger.log("Error opening file", "error");
             close(sockfd);
             return false;
         }
 
+        instance_logger.log("Sending", "info");
```

Comment:
There is no meaning for this log statement. Hence lets remove it.

`instance_logger.log("Sending", "info");`

Reply:
Removed it

Reply:
Ok